Failed to download action lowply/build-hugo

Hello, 
I found your GitHub action on deploying to firebase. But I was unable to successfully apply this action. It keeps on failing with the response that it was unable to download `lowply/build-hugo`
```
Current runner version: '2.165.2'
Prepare workflow directory
Prepare all required actions
Download action repository 'actions/checkout@master'
Download action repository 'lowply/build-hugo@v0.0.2'
##[warning]Failed to download action 'https://api.github.com/repos/lowply/build-hugo/tarball/v0.0.2'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 20.766 seconds before retry.
##[warning]Failed to download action 'https://api.github.com/repos/lowply/build-hugo/tarball/v0.0.2'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 13.996 seconds before retry.
##[error]Response status code does not indicate success: 404 (Not Found).
```

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ jobs:
     - name: Check out code
       uses: actions/checkout@master
     - name: Build Hugo
-      uses: lowply/build-hugo@v0.0.2
+      uses: lowply/build-hugo@v0.68.3
     - name: Deploy to Firebase
       uses: lowply/deploy-firebase@v0.0.2
       env:

